### ISSUE TITLE
Missing translation on "Review your activity"

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#1623]
## Describe the bug 

We missed a translation when logging in and reviewing log-in events.

This string can be added to fr.csv to fix the bug

```csv
"Review your activity","Examinez votre activité"
```

As defined in this translation reference in [Google Docs](https://docs.google.com/document/d/1Lg5GX8Fu8iPuVSDfMTKmMT2ICmh3cfjQFtAo0Q4sow8/edit)

## [Bug Severity](https://docs.google.com/document/d/1tcD32bjJRaREsMgaUEoXAk400y1dKzzaKR0G5Sr1w20/edit)
See examples in the documentation

SEV-4 while this only impacts staging. 

## To Reproduce

1. Go to Staging notify
2. Sign in
3. Get to the sign-in history
3. Switch to French
4. Notice missing translation. 

## Expected behavior

We should have a translation for every string.

## Impact

Content not consistent with the page's language

If applicable

Impact on Notify users: Could be difficult to understand if using SR, or unexpected for all. 

Impact on Recipients: None

Impact on Notify team: None

## Screenshots

<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN0lZQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--28175ffcd482134d46c836c2ff1893e67dde02c2/Capture%20d%E2%80%99e%CC%81cran%2C%20le%202024-07-29%20a%CC%80%2014.16.16.png" alt="Capture d’écran, le 2024-07-29 à 14.16.16.png" />

## Additional context

Should we have an automated check to catch missing translations? Right now, the app and tests do not fail, and this feels like a failure.
